src/__tests__/index.js

```diff
@@ -348,3 +348,34 @@ test('supports automatic reset of error boundary when resetKeys change', () => {
   expect(screen.queryByRole('alert')).not.toBeInTheDocument()
   expect(console.error).not.toHaveBeenCalled()
 })
+
+test('should support not only function as FallbackComponent', () => {
+  const container = document.createElement('div')
+  document.body.appendChild(container)
+  ;[React.forwardRef, React.memo, component => component].forEach(
+    componentCreator => {
```

Comment:
I don't think it's necessary to have all of these. Just one should give us plenty of confidence.